### PR TITLE
gci: Re-Add `TEST_NETWORK=liquid-regtest` to CI run

### PR DESCRIPTION
**Pull request by cdecker**
My bad for forgetting it in the Rework CI.

Changelog-None
Suggested-by: Greg Sanders <@instagibbs>

### CODE REVIEW

File: .github/workflows/ci.yaml [hunk 1]
```diff
@@ -178,39 +178,51 @@ jobs:
...
             CFG: gcc-dev1-exp1
             DEVELOPER: 1
             EXPERIMENTAL_FEATURES: 1
             COMPILER: gcc
             TEST_DB_PROVIDER: postgres
-
+            TEST_NETWORK: regtest
+          # And don't forget about elements (like cdecker did when
+          # reworking the CI...)
+          - NAME: gcc-dev1-exp0
```
Comment (instagibbs) [inline .github/workflows/ci.yaml:221]: maybe call this liquid-regtest then?

Comment (cdecker) [reply .github/workflows/ci.yaml]: That's what I get for "just doing this quickly before getting dinner" xD

**Comment by cdecker** [comment]
Added a small commit I'm going to remove afterwards, to make sure we actually end up testing `elementsd`.

**Comment by instagibbs** [comment]
Liquid tests definitely breaking

**Comment by vincenzopalazzo** [comment]
ACK https://github.com/ElementsProject/lightning/pull/5910/commits/51c436cf0db31714ce749b87c3502d3c0ae3958f

**Comment by endothermicdev** [comment]
Due to a large number of flakes that need addressing, pushing this until after 23.02 release.  The aim is to update libwally to 0.8.8, (PSBTv0 -> PSBTv2), then address any remaining flakes.

**Comment by instagibbs** [comment]
Would be appreciated if this is left open for tracking, as I can work on top of this(and compared failure notes)

**Comment by cdecker** [comment]
I'll mark it as `Draft` so we don't accidentally merge it ^^

**Comment by instagibbs** [comment]
I'm unable to replicate any of these failures locally unfortunately, with elements 0.18 or 22.0

Will continue investigating.

**Comment by instagibbs** [comment]
can be closed after https://github.com/ElementsProject/lightning/pull/5955 was merged

**Pull request closed without merging**